src/table/styles.scss

```diff
@@ -53,6 +53,7 @@
 
 .table {
   inline-size: 100%;
+  block-size: 100%;
```

Comment:
This change does not affect the table itself - the size of the table is determined by the size of its content.

However, the table element needs the height set for cells height to take an effect, see discussion: https://stackoverflow.com/questions/3215553/make-a-div-fill-an-entire-table-cell.